### PR TITLE
Add featured loaders for use on frontend

### PR DESCRIPTION
**Pull request by wafflecoffee**

### CODE REVIEW

File: src/database/models/categories.rs [hunk 1]
```diff
@@ -262,6 +263,7 @@ pub struct LoaderBuilder<'a> {
     pub name: Option<&'a str>,
     pub icon: Option<&'a str>,
     pub supported_project_types: Option<&'a [ProjectTypeId]>,
+    pub featured: &'a bool,
```
Comment (BasiqueEvangelist) [inline src/database/models/categories.rs:266]: Why are you using a reference here?

**Review by BasiqueEvangelist — COMMENTED** [review]
Did you run `cargo sqlx prepare`?

File: src/database/models/categories.rs (hunk 1 above)
Comment (wafflecoffee) [reply src/database/models/categories.rs]: Dunno, everything else was so I just copied it, removed it now

**Review by Geometrically — APPROVED** [review]

**Comment by wafflecoffee** [comment]
Closing as hardcoding in other places (namely proxies) has made this obsolete/outdated/not inclusive of all of our use cases.

**Comment by Prospector** [comment]
I'd suggest that this is still needed, headers would be a separate field I think (like categories have)

**Pull request closed without merging**